docs/ref/contrib/gis/gdal.txt

```diff
@@ -1111,9 +1111,9 @@ blue.
     raster should be opened in write mode. For newly-created rasters, the second
     parameter is ignored and the new raster is always created in write mode.
 
-    The first parameter can take three forms: a string representing a file
-    path, a dictionary with values defining a new raster, or a bytes object
-    representing a raster file.
+    The first parameter can take three forms: a string representing a file path
+    (filesystem or GDAL virtual filesystem), a dictionary with values defining
+    a new raster, or a bytes object representing a raster file.
```

Comment:
Maybe add an example. The VSI functionality has gained significant traction lately, so adding an example might be relevant.

Reply:
Thanks @yellowcap , as suggested by @felixxm I'll update the docs in [Using GDALs virtual filesystem](https://docs.djangoproject.com/en/3.2/ref/contrib/gis/gdal/#using-gdal-s-virtual-filesystem), so instead of just pointing it out in a small parentheses I think I'll add two subsections there.

* In Memory Raster -> driver=Mem, current documentation.
* Other virtual filesystems -> not specifically supported via driver but just sending the vsi paths. Some examples and maybe a link to the GDAL documentation

Reply:
Sounds good!